### PR TITLE
fix: 只在移动端绑定tap事件，防止多次触发 collapse 事件

### PR DESCRIPTION
**Pull request by wjgogogo**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/src/cell/row-cell.ts [hunk 1]
```diff
@@ -139,16 +143,18 @@ export class RowCell extends HeaderCell {
...
+      this.gm = new GM(this, {
+        gestures: ['Tap'],
       });
-    });
+      this.gm.on('tap', () => {
+        this.spreadsheet.emit(S2Event.ROW_CELL_COLLAPSE_TREE_ROWS, {
+          id,
+          isCollapsed: !isCollapsed,
+          node: this.meta,
+        });
```
Comment (stone-lyl) [inline packages/s2-core/src/cell/row-cell.ts:155]: 这个部分事件绑定重复了，可以提取成一个方法

Comment (wjgogogo) [reply packages/s2-core/src/cell/row-cell.ts:155]: 这部分就是一个方法调用，只有参数有点多😰

**Review by lijinke666 — APPROVED** [review]

File: packages/s2-core/src/utils/g-renders.ts [hunk 2]
```diff
@@ -88,14 +88,16 @@ export function renderTreeIcon(
...
-  onClick: () => void,
+  onClick?: () => void,
 ) {
   const icon = new GuiIcon({
     name: isCollapse ? 'Plus' : 'Minus',
     ...area,
     fill,
   });
-  icon.on('click', onClick);
+  if (onClick) {
```
Comment (xingwanying) [inline packages/s2-core/src/utils/g-renders.ts:98]: if(isFunction(onClick))

**Review by lijinke666 — APPROVED** [review]

**Review by xingwanying — APPROVED** [review]

**Pull request merged**